CHANGELOG.md

```diff
@@ -8,18 +8,34 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 - Support for `idempotencyKey` for `POST` methods by @aydrian & @rileylnapier
+- Support for [Lists API](https://docs.courier.com/reference/lists-api) by @aydrian
+  - `POST /send/list` via `client.lists.send(params, config)`
+  - `GET /profiles/{recipient_id}/lists` via `client.lists.findByRecipientId(recipientId, params)`
+  - `GET /lists` via `client.lists.list(params)`
+  - `GET /lists/{list_id}` via `client.lists.get(listId)`
+  - `PUT /lists/{list_id}` via `client.lists.put(listId, {...})`
+  - `DELETE /lists/{list_id}` via `client.lists.delete(listId)`
+  - `PUT /lists/{list_id}/restore` via `client.lists.restore(listId)`
+  - `GET /lists/{list_id}/subscriptions` via `client.lists.getSubscriptions(listId)`
+  - `POST /lists/{list_id}/subscriptions` via `client.lists.bulkSubscribe(listId, [recipientId], config)`
```

Comment:
i know this was our suggestion, but wanted to throw another option after getting some feedback yesterday: `putSubscriptions` instead of `bulkSubscribe`. i believe that will make it more clear that this call creates or replaces the list of subscriptions. 

thoughts @aydrian @trycourier/engineering?

Reply:
I wait to see what everyone thinks

Reply:
`bulkSubscribe` isn't actually accurate here (and it's a PUT to `/subscribers`) - this is an idempotent overwrite of the subscribers list

Reply:
So what do we need to change? Or do we?

Reply:
My suggestion would be `putSubscribers` or `replaceSubscribers` (I'm fine interchanging `subscriptions/subscribers`)